serverless.example.yml

```diff
@@ -101,9 +126,30 @@ functions:
       - http:
           path: beacons/volatility
           method: get
+          cors:
+            headers: '*'
+            origin: 'http://domain-one.com,https://domain-two.com'
+            methods:
+              - OPTIONS
+              - GET
           caching:
-            enabled: true
+            enabled: false
```

Comment:
Why disable caching here?

Reply:
This is a mistake, thank you, I'll fix it.